moto/secretsmanager/models.py

```diff
@@ -592,6 +651,54 @@ def untag_resource(self, secret_id, tag_keys):
 
         return secret_id
 
+    def update_secret_version_stage(
+        self, secret_id, version_stage, remove_from_version_id, move_to_version_id
+    ):
+        if secret_id not in self.secrets.keys():
+            raise SecretNotFoundException()
+
+        secret = self.secrets[secret_id]
+
+        if remove_from_version_id:
+            if remove_from_version_id not in secret.versions:
+                raise InvalidParameterException(
```

Comment:
Thanks for the tests @danielsamuels. Ideally we have some tests for these error scenarios as well, using the regular boto3 Client.
Example on how we do these negative tests in moto can be found here: https://github.com/spulec/moto/blob/master/tests/test_secretsmanager/test_list_secrets.py#L133

Reply:
Added in 5c51a48